src/markdown-pages/contribute-to-quickstarts/instrument-library/send-metrics.mdx

```diff
@@ -0,0 +1,734 @@
+    stats[f"{type_}_errors"] = 0
+    stats["cache_hit"] = 0
+    stats[f"{type_}_count"] = 0
+    last_push[type_] = datetime.datetime.now()
+
+```
+
+</Project>
+
+## Familiarize yourself with the application
+
+Open _db.py_ file in the IDE of your choice and familiarize yourself with the app logic.
+
+This demo uses a dummy Python application that mimics the Create, Read, Update, and Delete (CRUD) operations.
+
+<>
+
+```python fileName=db.py
+import os
+import random
+import datetime
+
+db = {}
+stats = {
+    "read_response_times": [],
+    "read_errors": 0,
+    "read_count": 0,
+    "create_response_times": [],
+    "create_errors": 0,
+    "create_count": 0,
+    "update_response_times": [],
+    "update_errors": 0,
+    "update_count": 0,
+    "delete_response_times": [],
+    "delete_errors": 0,
+    "delete_count": 0,
+    "cache_hit": 0,
+}
+last_push = {
+    "read": datetime.datetime.now(),
+    "create": datetime.datetime.now(),
+    "update": datetime.datetime.now(),
+    "delete": datetime.datetime.now(),
+}
+
+def read(key):
+
+    print(f"Reading...")
+
+    if random.randint(0, 30) > 10:
+        stats["cache_hit"] += 1
+
+    stats["read_response_times"].append(random.uniform(0.5, 1.0))
+    if random.choice([True, False]):
+        stats["read_errors"] += 1
+    stats["read_count"] += 1
+    try_send("read")
+
+def create(key, value):
+
+    print(f"Writing...")
+
+    db[key] = value
+    stats["create_response_times"].append(random.uniform(0.5, 1.0))
+    if random.choice([True, False]):
+        stats["create_errors"] += 1
+    stats["create_count"] += 1
+    try_send("create")
+
+def update(key, value):
+
+    print(f"Updating...")
+
+    db[key] = value
+    stats["update_response_times"].append(random.uniform(0.5, 1.0))
+    if random.choice([True, False]):
+        stats["update_errors"] += 1
+    stats["update_count"] += 1
+    try_send("update")
+
+def delete(key):
+
+    print(f"Deleting...")
+
+    db.pop(key, None)
+    stats["delete_response_times"].append(random.uniform(0.5, 1.0))
+    if random.choice([True, False]):
+        stats["delete_errors"] += 1
+    stats["delete_count"] += 1
+    try_send("delete")
+
+def try_send(type_):
+
+    print("try_send")
+
+def clear(type_):
+    stats[f"{type_}_response_times"] = []
+    stats[f"{type_}_errors"] = 0
+    stats["cache_hit"] = 0
+    stats[f"{type_}_count"] = 0
+    last_push[type_] = datetime.datetime.now()
+
+```
+</>
+
+The `read`, `create`, `update`, and `delete` are the dummy methods to mimic CRUD operations. For every CRUD operations, respective `stat` is incremented to reflect that the operation has been performed. Next, you send this `stats` data to New Relic.
+
+## Send metrics to New Relic
+
+There are 3 different types of metrics:
+- GaugeMetric: It sends a single value at a single point in time.
+- CountMetric: It tracks the total number of occurrences of an event.
+- SummaryMetric: It tracks count, sum, min, and max values over time.
```

Comment:
You might remove the 'it' in these statements, I think it would flow a little easier.